devel/osc/Portfile

```diff
@@ -21,17 +21,16 @@ long_description    osc is a subversion-like client written in Python. \
 
 homepage            https://en.opensuse.org/openSUSE:OSC
 
-checksums           rmd160  c3915c31af985c645f902e9f9be840c24c0d297d \
-                    sha256  668d319c5ead8d879120c27638925e7476ccafa1144c047b56ccef10b0dd2689 \
-                    size    340944
+checksums           rmd160  333dff85a056f9f09298a1fdf660fb1795f20d7d \
+                    sha256  447158eed1252227315902af14a58f94ae17863c2db18f7fbfa80e214d0c25b1 \
+                    size    360443
 
 patchfiles          patch-config.diff \
                     patch-build-disable.diff
 
-python.default_version 27
+python.default_version 39
```

Comment:
Perhaps this should be `38` as the default Python version is still set to Python 3.8 within the `python` portgroup:
https://github.com/macports/macports-ports/blob/935058f1fea40c1cde63f7b45206acf60ab75ec9/_resources/port1.0/group/python-1.0.tcl#L74

Reply:
@herbygillot at another hand I can remove default version and use that macports use as default version.

I feel that it is much better than selected specified version like `38` or `34`.

Do you agree?

Reply:
It should be specifically set in order to have repeatable builds. Please use 38.

Reply:
@mf2k, @herbygillot fixed